Cargo.toml

```diff
@@ -27,5 +27,5 @@ hex-literal = "0.1"
 
 [features]
 default = []
-std = ["pwasm-std/std"]
+std = ["pwasm-std/std", "byteorder/std"]
```

Comment:
used some trait from, right?

Reply:
It is used in `src/eth/log.rs`

Reply:
`(*self).into()` - this one?

Reply:
More like here: https://github.com/paritytech/pwasm-abi/blob/master/src/eth/log.rs#L3

Reply:
ok, cool!

> used some trait from, right?

I was confused cause [it](https://github.com/paritytech/pwasm-abi/pull/77/files#diff-d10a55066c14f9ee9e39aff71f440d6aR3) was not a part of the PR :)